test/unit/webrtc.test.ts

```diff
@@ -769,38 +804,51 @@ describe('sendStats', () => {
     const webrtc = new WebrtcExtension(client as any);
 
     const sendSpy = jest.spyOn(client.http, 'requestApi').mockResolvedValue(null);
-
-    webrtc['statsToSend'].push({} as any);
+    webrtc['statsArr'].push({} as any);
     webrtc['throttledSendStats']();
     expect(sendSpy).not.toHaveBeenCalled();
-    await wait(3050);
+    await wait(25050);
```

Comment:
we should probably modify this. I think there's value is actually testing against a real timer, but adding a 25 sec delay in a test is pretty gnarly. Here, we probably want to use jest's fake timers. I think there's also a restoreTimers fn or something equivalent, so I'd start the test with fake timers, then restore at the end of the test.

Reply:
There are some example usages of fake timers in `test/unit/ping.test.ts` and `test/unit/reconnector.test.ts`. Just be careful if you start mixing them with promises. Things can get messy and confusing. I've spent a couple days working through fake timers and promise issues in tests.

Reply:
In previous versions of Jest there was some weird behavior with mock timers and debounce/throttle but version 26+ seems to have resolved it. I've updated the tests with mock timers.